src/application/extension-popup-menu/footer.tsx

```diff
@@ -0,0 +1,46 @@
+import { Icon, IconName } from 'shared/ui';
+
+interface FooterLink {
+  iconName: IconName;
+  href: string;
+  text: string;
+}
+
+export const Footer = () => {
```

Comment:
I suggest to create internal folder `components` and hold footer there to make it consistent with other places